npm/angular/src/mount.ts

```diff
@@ -301,6 +329,7 @@ function setupComponent<T extends { ngOnChanges? (changes: SimpleChanges): void
  * })
  * @returns Cypress.Chainable<MountResponse<T>>
  */
+// eslint-disable-next-line no-redeclare
```

Comment:
@yjaaidi , @jordanpowell88, @ZachJW34, I need your help, please.

I used function overloading to exclude `componentProviders` when a template is used. Now the original JSDoc description for the `mount` function doesn't show up in the IDE anymore. Any suggestions on how to fix that?